### PR TITLE
dw - 6pm-4-add a login count on view logins page

### PR DESCRIPTION
**Pull request by dorinori**
Add a login count on the view logins page so admins can see the number of times people have logged in to the web application.

### CODE REVIEW

**Review by bzamora020 — DISMISSED** [review]
LGTM!!

File: javascript/src/main/pages/Admin/ViewLogins.js [hunk 1]
```diff
@@ -25,10 +25,15 @@ const ViewLogins = () => {
   const { getAccessTokenSilently: getToken } = useAuth0();
   const { data:loginTable } = useSWR(["/api/admin/logins", getToken], fetchWithToken);
 
+  if (loginTable){
+    var size = loginTable.length;
+  }
+
```
Comment (andrewhlu) [inline javascript/src/main/pages/Admin/ViewLogins.js:31]: Is there a reason why `size` needs to be declared as an extra variable? Can we just access `loginTable.length` directly from the render code? IMO it looks nicer this way.

**Review by andrewhlu — CHANGES_REQUESTED** [review]

File: javascript/src/main/pages/Admin/ViewLogins.js (hunk 1 above)
Comment (dorinori) [reply javascript/src/main/pages/Admin/ViewLogins.js:31]: loginTable is undefined in some instances, is there any way for my to use .length on an undefined object?

Comment (andrewhlu) [reply javascript/src/main/pages/Admin/ViewLogins.js:31]: No, but you can the check for whether it exists inline using ternary operators.
Alternatively, you could logical AND the `<p>` tag and `loginTable` to only show the "Number of logins:" text when the data is loaded.

**Review by andrewhlu — APPROVED** [review]
Looks nice and clean :)

**Review by bzamora020 — APPROVED** [review]
LGTM!

**Pull request merged**